src/pages/haiinkim/Main/Main.js

```diff
@@ -0,0 +1,318 @@
+import React from "react";
+import "./Main.scss";
+import List from "./List";
+//import COMMENT from "./commentData";
+import { IoLogoInstagram } from "react-icons/io";
+import { BiSearch } from "react-icons/bi";
+import { AiFillHome } from "react-icons/ai";
+import { RiSendPlaneLine } from "react-icons/ri";
+import { AiOutlineCompass } from "react-icons/ai";
+import { AiOutlineHeart } from "react-icons/ai";
+import { BiComment } from "react-icons/bi";
+import { BiBookmark } from "react-icons/bi";
+import { BsThreeDots } from "react-icons/bs";
+
+const USER = "Haiin";
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputComment: "",
+      commentList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/data.json", {
+      method: "GET",
+    })
```

Comment:
```suggestion
    fetch("http://localhost:3000/data/data.json")
```
- default method가 "GET"이기 때문에 이렇게도 할 수 있습니다.

Reply:
넵 생략하겠습니다.